faunadb/client.py

```diff
@@ -187,6 +189,31 @@ def query(self, expression, timeout_millis=None):
     """
     return self._execute("POST", "", _wrap(expression), with_txn_time=True, query_timeout_ms=timeout_millis)
 
+  def stream(self, expression, options=None, on_start=None, on_error=None, on_version=None, on_history=None, blocking=True):
```

Comment:
Maybe use kwargs for options and event handlers?

Reply:
I would lean more on not using kwargs since we have a finite and well defined set of possible events and names of event handlers. IMO it also improves readability and documentation.